components/services/bat_ads/bat_ads_client_mojo_bridge.cc

```diff
@@ -96,6 +96,15 @@ bool BatAdsClientMojoBridge::IsEnabled() const {
   return is_enabled;
 }
 
+bool BatAdsClientMojoBridge::CanShowBackgroundNotifications() const {
+  if (!connected())
```

Comment:
nit: please add curly braces for consistency with rest of file, i.e.
```
if (!connected()) {
    return false;
}
```

Reply:
NONCRITICAL
Covered by: brave/brave-browser#6235 (15)